zanata-war/src/main/java/org/zanata/action/VersionHome.java

```diff
@@ -370,7 +370,6 @@ public String createVersion() {
     }
 
     public void copyVersion() {
-        getInstance().setStatus(EntityStatus.READONLY);
```

Comment:
I thought we mark it as read only before the copy is finished. Has this changed?